configs/containers/specs/jedi-ci.yaml

```diff
@@ -2,10 +2,9 @@
   specs: [base-env@1.0.0, jedi-base-env@1.0.0, ewok-env@1.0.0, jedi-fv3-env@1.0.0,
     jedi-mpas-env@1.0.0, bacio@2.4.1, bison@3.8.2, bufr@12.0.1, ecbuild@3.7.2, eccodes@2.33.0, ecflow@5,
     eckit@1.24.5, ecmwf-atlas@0.36.0 +fckit +trans +tesselation +fftw, fiat@1.2.0, ectrans@1.2.0 +fftw,
-    eigen@3.4.0, fckit@0.11.0, fms@2023.04, g2@3.4.9, g2tmpl@1.10.2, gftl-shared@1.6.1,
+    eigen@3.4.0, fckit@0.11.0, fms@2023.04, g2@3.4.9, g2tmpl@1.10.2,
     gsibec@1.2.1, hdf@4.2.15, hdf5@1.14.3, ip@5.0.0, jasper@2.0.32, jedi-cmake@1.4.0,
-    libpng@1.6.37, nccmp@1.9.0.1, netcdf-c@4.9.2, netcdf-cxx4@4.3.1,
-    netcdf-fortran@4.6.1, nlohmann-json@3.10.5, nlohmann-json-schema-validator@2.1.0,
+    libpng@1.6.37, nccmp@1.9.0.1, netcdf-c@4.9.2, netcdf-cxx4@4.3.1, netcdf-fortran@4.6.1,
```

Comment:
Doesn't the removal of the `nlohmann-json` and `nlohmann-json-schema-validator` packages disable the validation of all the YAML configuration in JEDI? Or are these packages still getting picked up as dependencies in the build? Just want to make sure that all of the oops::Parameters::validateAndDeserialize() function calls are still validating.

Reply:
nlohmann-json are dependencies of jedi-fv3-env, which is listed above. This just removes the hardcoded versions.

Reply:
Great! Thanks for the explanation @climbfuji